docs/queries/filters.md

```diff
@@ -39,54 +53,75 @@ Natural language notation:
 - `last friday`
 - `14 days ago`
 - `in two weeks`
+- `14 October` (the current year will be used)
+- `May` (1st May in the current year will be used)
 
 Note that if it is Wednesday and you write `tuesday`, Tasks assumes you mean "yesterday", as that is the closest Tuesday.
```

Comment:
Does `this tuesday` point to this week's Tue is today is Wed? Just curious, maybe it is worth a sentence here

Reply:
Sorry, I don't understand the point. I think that it is what it is already trying to say?

Reply:
Oops I was referencing the line 59. He point is to force writing `this tuesday` instead of `tuesday`

Reply:
Oh, you are probably right, but this is text I haven't changed, and I'm running out of time on this! 😢 

So thank you for the suggestion, and sorry, but I'll leave this one as it is.